phpBB/viewforum.php

```diff
@@ -782,9 +782,11 @@
 * @var	array	topic_list			Array with current viewforum page topic ids
 * @var	array	rowset				Array with topics data (in topic_id => topic_data format)
 * @var	int		total_topic_count	Forum's total topic count
+* @var	int		forum_id			Forum identifier
 * @since 3.1.0-b3
+* @change 3.1.11-RC1 Added forum_id
```

Comment:
Should be `@changed` as discussed in the PR for changing `@change`  to `@changed`